stubs/python-xlib/Xlib/error.pyi

```diff
@@ -0,0 +1,76 @@
+from _typeshed import SliceableBuffer
+from typing_extensions import Literal
+
+from Xlib.protocol import display, rq
+
+class DisplayError(Exception):
+    display: object
+    def __init__(self, display: object) -> None: ...
+
+class DisplayNameError(DisplayError): ...
+
+class DisplayConnectionError(DisplayError):
+    display: object
+    msg: object
+    def __init__(self, display: object, msg: object) -> None: ...
+
+class ConnectionClosedError(Exception):
+    whom: object
+    def __init__(self, whom: object) -> None: ...
+
+class XauthError(Exception): ...
+class XNoAuthError(Exception): ...
+class ResourceIDError(Exception): ...
+
+class XError(rq.GetAttrData, Exception):
+    def __init__(self, display: display.Display, data: SliceableBuffer) -> None: ...
+
+class XResourceError(XError): ...
+class BadRequest(XError): ...
+class BadValue(XError): ...
+class BadWindow(XResourceError): ...
+class BadPixmap(XResourceError): ...
+class BadAtom(XError): ...
+class BadCursor(XResourceError): ...
+class BadFont(XResourceError): ...
+class BadMatch(XError): ...
+class BadDrawable(XResourceError): ...
+class BadAccess(XError): ...
+class BadAlloc(XError): ...
+class BadColor(XResourceError): ...
+class BadGC(XResourceError): ...
+class BadIDChoice(XResourceError): ...
+class BadName(XError): ...
+class BadLength(XError): ...
+class BadImplementation(XError): ...
+
+xerror_class: dict[
+    int,
+    type[BadRequest]
+    | type[BadValue]
+    | type[BadWindow]
+    | type[BadPixmap]
+    | type[BadAtom]
+    | type[BadCursor]
+    | type[BadFont]
+    | type[BadMatch]
+    | type[BadDrawable]
+    | type[BadAccess]
+    | type[BadAlloc]
+    | type[BadColor]
+    | type[BadGC]
+    | type[BadIDChoice]
+    | type[BadName]
+    | type[BadLength]
+    | type[BadImplementation],
```

Comment:
Indeed seems like a valid simplification in the stubs. Since int keys can't be used with TypedDicts anyway, it's not like I can represent the exact value at runtime perfectly.